mailpoet/lib/Config/Menu.php

```diff
@@ -401,51 +420,56 @@ private function registerMailPoetMenu() {
     $this->wp->addSubmenuPage(
       true,
-      $this->setPageTitle('Logs'),
+      $this->setPageTitle(__('Logs', 'mailpoet')),
       '',
       AccessControl::PERMISSION_ACCESS_PLUGIN_ADMIN,
       'mailpoet-logs',
       [$this, 'logs']
     );
 
-    // Automation
-    if ($this->featuresController->isSupported(FeaturesController::AUTOMATION)) {
-      $this->wp->addSubmenuPage(
-        self::MAIN_PAGE_SLUG,
-        $this->setPageTitle('Automation'),
-        'Automation',
-        AccessControl::PERMISSION_MANAGE_EMAILS,
-        'mailpoet-automation',
-        [$this, 'automation']
-      );
-
-      // Automation editor
-      $automationEditorPage = $this->wp->addSubmenuPage(
-        true,
-        $this->setPageTitle('Automation Editor'),
-        'Automation Editor',
-        AccessControl::PERMISSION_MANAGE_AUTOMATIONS,
-        'mailpoet-automation-editor',
-        [$this, 'automationEditor']
-      );
-
-      // Automation templates
-      $this->wp->addSubmenuPage(
-        true,
-        $this->setPageTitle('Automation Templates'),
-        'Automation Templates',
-        AccessControl::PERMISSION_MANAGE_AUTOMATIONS,
-        'mailpoet-automation-templates',
-        [$this, 'automationTemplates']
-      );
-
-      // add body class for automation editor page
-      $this->wp->addAction('load-' . $automationEditorPage, function() {
-        $this->wp->addAction('admin_body_class', function ($classes) {
-          return ltrim($classes . ' site-editor-php');
-        });
-      });
+  }
+
+  private function registerAutomationMenu() {
+
+    if (!$this->featuresController->isSupported(FeaturesController::AUTOMATION)) {
+      return;
     }
+    $this->wp->addSubmenuPage(
+      self::MAIN_PAGE_SLUG,
+      $this->setPageTitle(__('Automations', 'mailpoet')),
+      // @ToDo Remove Beta once Automation is no longer beta.
+      '<span>' . esc_html__('Automations', 'mailpoet') . '</span><span class="mailpoet-beta-badge">Beta</a>',
+      AccessControl::PERMISSION_MANAGE_EMAILS,
+      'mailpoet-automation',
+      [$this, 'automation']
+    );
+
+    // Automation editor
+    $automationEditorPage = $this->wp->addSubmenuPage(
+      true,
+      $this->setPageTitle(__('Automation Editor', 'mailpoet')),
+      esc_html__('Automation Editor', 'mailpoet'),
+      AccessControl::PERMISSION_MANAGE_AUTOMATIONS,
+      'mailpoet-automation-editor',
+      [$this, 'automationEditor']
+    );
+
+    // Automation templates
+    $this->wp->addSubmenuPage(
+      true,
+      $this->setPageTitle(__('Automation Templates', 'mailpoet')),
+      esc_html__('Automation Templates', 'mailpoet'),
+      AccessControl::PERMISSION_MANAGE_AUTOMATIONS,
+      'mailpoet-automation-templates',
+      [$this, 'automationTemplates']
+    );
+
+    // add body class for automation editor page
+    $this->wp->addAction('load-' . $automationEditorPage, function() {
+      $this->wp->addAction('admin_body_class', function ($classes) {
```

Comment:
I know this was already here and you just switched to using an early return, but I think this should be `addFilter` instead of `addAction`.

Reply:
Good catch. Addressed in 677c9ee19